packages/api/endpoints/execution-status.js

```diff
@@ -10,14 +10,14 @@ const { StepFunction } = require('@cumulus/ingest/aws');
 function get(event, cb) {
   const arn = _get(event.pathParameters, 'arn');
 
-  StepFunction.getExecutionStatus(arn)
+  return StepFunction.getExecutionStatus(arn)
```

Comment:
Was this causing a bug? (If yes what was the bug?)

Reply:
yeah, if you return a function that has promise inside, you have to make sure that all the other functions with promise also return a promise or the program exists prematurely.

Reply:
👍